execution_workflows/diffUTR/diffUTR_nextflow/build_docker_image.md

```diff
@@ -0,0 +1,9 @@
+## Build Docker image from this directory with enviroment.yml and Dockerfile
+```
+docker build . -t [execution_tool]:1.0
```

Comment:
Can this be used inside `diffUTR_nextflow`? If yes, fill in correct names, if no, remove file from directory.